data/records/alice-2010-pbpb.json

```diff
@@ -0,0 +1,69 @@
+[
+  {
+    "abstract": {
+      "description": "Lead-Lead data sample at the collision energy of 2.76TeV from run number 137135 of the LHC10h data taking period."
+    },
+    "accelerator": "CERN-LHC",
+    "collaboration": {
+      "name": "ALICE Collaboration"
+    },
+    "collections": [
+      "ALICE-Reconstructed-Data"
+    ],
+    "collision_information": {
+      "energy": "2.76TeV",
+      "type": "PbPb"
+    },
+    "created": "2025-02-11T15:53:44.819508",
+    "date_created": [
+      "2010"
+    ],
+    "date_published": "2025",
+    "distribution": {
+      "formats": [
+        "root"
+      ],
+      "number_events": 213750,
+      "number_files": 128,
+      "size": 2641170373
+    },
+    "experiment": [
+      "ALICE"
+    ],
+    "files": [
+      {
+        "checksum": "adler32:7ac174b9",
+        "size": 27508,
+        "type": "index.json",
+        "uri": "root://eospublic.cern.ch//eos/opendata/alice/upload/2010/LHC10h/000139510/file-indexes/LHC10h_000139510_PbPb_2.76TeV_205656_file_index.json"
+      }
+    ],
+    "license": {
+      "attribution": "CC0-1.0"
+    },
+    "publisher": "CERN Open Data Portal",
+    "recid": "1114",
+    "title": "LHC10h_000137135_PbPb_2.76TeV_213750",
+    "title_additional": "Lead-Lead data sample at the collision energy of 2.76TeV from run number 137135 of the LHC10h data taking period",
```

Comment:
For the former ALICE records, the description used to say "Pb-Pb data sample" instead of "lead". Another cosmetic observation for harmonising.

Reply:
Ok, let's get back to symbolic name. How should pp look in this phrase? `Pp`, `PP`, or `pp` ?